### PR TITLE
Document `order:`

### PR DESCRIPTION
**Pull request by davidmanzanares**
Fix https://github.com/CartoDB/carto-vl/issues/20

### CODE REVIEW

File: examples/styling/draw-order.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Draw order | CARTO</title>
+  <title>raw order  | CARTO</title>
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Draw

Comment (davidmanzanares) [reply examples/styling/draw-order.html]: Ooops, copy paste problems :joy:

File: examples/styling/draw-order.html [hunk 2]
```diff
@@ -14,22 +14,40 @@
...
+      <div class="box">
+        <header>
+          <h1>Color schemes</h1>
+        </header>
+        <section>
+          <p class="description open-sans">How to use CARTOcolors and custom color schemes</p>
+          <div id="controls">
+            <ul>
+              <li>
+                <input type="radio" name="style" onclick="setAscOrder()" id="Cartocolor" checked>
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Change id to `Asc`

File: examples/styling/draw-order.html [hunk 3]
```diff
@@ -14,22 +14,40 @@
...
-      <li>
-        <input type="radio" name="style" onclick="setDescOrder()" id="des">
-        <label for="des">Descending</label>
-      </li>
-    </ul>
-  </div>
+    <aside class="toolbox">
+      <div class="box">
+        <header>
+          <h1>Color schemes</h1>
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Change title

File: examples/styling/draw-order.html [hunk 4]
```diff
@@ -14,22 +14,40 @@
...
+        </header>
+        <section>
+          <p class="description open-sans">How to use CARTOcolors and custom color schemes</p>
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Change description

File: examples/styling/draw-order.html [hunk 5]
```diff
@@ -14,22 +14,40 @@
...
+          <div id="controls">
+            <ul>
+              <li>
+                <input type="radio" name="style" onclick="setAscOrder()" id="Cartocolor" checked>
+                <label for="Asc">Asc</label>
+              </li>
+              <li>
+                <input type="radio" name="style" onclick="setDescOrder()" id="CartocolorReverse">
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Change id to `Desc`

File: examples/styling/draw-order.html [hunk 6]
```diff
@@ -14,22 +14,40 @@
...
+                <label for="Desc">Desc</label>
+              </li>
+              <li>
+                <input type="radio" name="style" onclick="setDefaultOrder()" id="CustomColors">
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Change id to `Default`

File: examples/styling/draw-order.html [hunk 7]
```diff
@@ -14,22 +14,40 @@
...
+      <div class="box">
+        <header>
+          <h1>Draw Order</h1>
+        </header>
+        <section>
+          <p class="description open-sans">How to use CARTO ordering style property</p>
+          <div id="controls">
+            <ul>
+              <li>
+                <input type="radio" name="style" onclick="setAscOrder()" checked>
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: Please keep the `id` to allow changing the radio button by clicking the text (that's why the for="" exists)

File: examples/styling/draw-order.html [hunk 8]
```diff
@@ -14,22 +14,40 @@
...
+                <label for="Asc">Asc</label>
+              </li>
+              <li>
+                <input type="radio" name="style" onclick="setDescOrder()">
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: https://github.com/CartoDB/carto-vl/pull/884/files#r211992972

File: examples/styling/draw-order.html [hunk 9]
```diff
@@ -14,22 +14,40 @@
...
+                <label for="Desc">Desc</label>
+              </li>
+              <li>
+                <input type="radio" name="style" onclick="setDefaultOrder()">
```
Comment (Jesus89) [inline examples/styling/draw-order.html]: https://github.com/CartoDB/carto-vl/pull/884/files#r211992972

**Pull request merged**